graphbolt/src/csc_sampling_graph.cc

```diff
@@ -95,6 +95,39 @@ void CSCSamplingGraph::Save(torch::serialize::OutputArchive& archive) const {
   }
 }
 
+void CSCSamplingGraph::SetState(
+    std::vector<torch::Dict<std::string, torch::Tensor>>& state) {
+  TORCH_CHECK(state.size() >= 1 && state.size() <= 2, "Wrong length of state.");
```

Comment:
why the magic number is 1 or 2? Maybe add a comment?

Reply:
It's not magic number but the length of the vector. I've added some comments.

Reply:
It depends on how we define the serialized data structure. I think it is better to use `torch::Dict<std::string, torch::Dict<std::string, torch::Tensor>>`, where the keys of the outer `Dict` are something like `regular_tensor` and `edge_attribute`.